automation/tests/organizationGroupTests.test.js

```diff
@@ -146,31 +147,44 @@ test.describe('Organization Group Tx tests', () => {
     expect(secondResult).toBe('SUCCESS');
   });
 
-  //This test (for the 5 transactions option) appears to be a bit flaky, but very rare.
-  [5, 100].forEach((numberOfTransactions) => {
-    test(`Verify user can import csv transactions with ${numberOfTransactions} transactions`, async () => {
-      test.slow();
-      await groupPage.fillDescription('test');
-      await groupPage.generateAndImportCsvFile(complexKeyAccountId, newAccountId,  numberOfTransactions);
-      const message = await groupPage.getToastMessage();
-      expect(message).toBe('Import complete');
-      await groupPage.clickOnSignAndExecuteButton();
-      await groupPage.clickOnConfirmGroupTransactionButton();
-      const timestamps = await groupPage.getAllTransactionTimestamps(numberOfTransactions);
-      await groupPage.clickOnSignAllButton();
-      await loginPage.waitForToastToDisappear();
-      await transactionPage.clickOnTransactionsMenuButton();
-      await organizationPage.logoutFromOrganization();
-      await groupPage.logInAndSignGroupTransactionsByAllUsers(globalCredentials.password, numberOfTransactions > 10);
-      await organizationPage.signInOrganization(
-        firstUser.email,
-        firstUser.password,
-        globalCredentials.password,
-      );
-      const isAllTransactionsSuccessful =
-        await groupPage.verifyAllTransactionsAreSuccessful(timestamps);
-      expect(isAllTransactionsSuccessful).toBe(true);
-    });
+  test('Verify user can cancel all items in a transaction group', async () => {
+    test.slow();
+    await groupPage.addSingleTransactionToGroup(2);
+
+    await groupPage.clickOnSignAndExecuteButton();
+    await groupPage.clickOnConfirmGroupTransactionButton();
+    await groupPage.clickOnCancelAllButton();
+    await groupPage.clickOnConfirmGroupActionButton()
+    await loginPage.waitForToastToDisappear();
+    await transactionPage.clickOnTransactionsMenuButton();
+    await organizationPage.clickOnReadyToSignTab()
+    expect(await groupPage.isEmptyTransactionTextVisible()).toBe(true);
+  });
+
+  test(`Verify user can import csv with 100 transactions`, async () => {
```

Comment:
This test was set to run with 5 and 100 because it was going through all the signing of the transactions using the 2 different flows: sign all and sign each individually. Do we still test the 'sign individually' flow?